src/main/java/org/kie/jenkinsci/plugins/kieprbuildshelper/RepositoryLists.java

```diff
@@ -70,7 +64,9 @@ private static URL createUrlForRepositoryList(GitHubRepository repo, GitBranch b
      * TODO: this is an ugly hack. The dependency between repositories (or directly modules) should to be checked automatically for every build
      */
     public static List<Tuple<GitHubRepository, GitBranch>> filterOutUnnecessaryRepos(List<Tuple<GitHubRepository, GitBranch>> repos, GitHubRepository baseRepo) {
-        // nothing depends on stuff from -tools repo
+        // Errai is not upstream repository anymore
+    	repos.removeIf(repo -> repo._1().equals(new GitHubRepository("errai", "errai")));
```

Comment:
Is this needed? I think after removing the `branch-mapping.yaml` file the only source for repos is the droolsjbpm-build-`bootstrap/scripts/repository-list.txt` file where `errai` is not listed

Reply:
no, that was some tryout ;)